### DISCUSSION TITLE
uBlock instructions unclear

### DISCUSSION THREAD

**Discussion by weenslurper** [3kh0/ext-remover#757] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE]
step 4 says "Inside the advanced settings, change userResourcesLocation from unset to https://pastebin.com/raw/PPZ1T7uh (image 2)"

do I paste the code in or just the link? neither of them worked for me, and there's also no image to go with it so it's kinda unclear

**Comment by S-PScripts** [comment]
the code.

make sure you are following instructions right: https://github.com/S-PScripts/chromebook-utilities/blob/main/Exploits/Run%20Code%20(with%20uBlock)

(necropost 2 answer)
